ingest/src/main/java/com/connexta/ingest/adaptors/S3Adaptor.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.ingest.adaptors;
+
+import com.connexta.ingest.service.api.IngestRequest;
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.util.UUID;
+import javax.annotation.PostConstruct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.AwsCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.core.exception.SdkClientException;
+import software.amazon.awssdk.core.sync.RequestBody;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.PutObjectRequest;
+import software.amazon.awssdk.services.s3.model.S3Exception;
+
+@Service
+public class S3Adaptor implements Adaptor {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(S3Adaptor.class);
+  private S3Client s3Client;
+
+  @Autowired private String s3BucketQuarantine;
+
```

Comment:
I think typically using `@Autowired` on the constructor and using setters is safer than inlining it here.